docs/en/guide/cache_management.md

```diff
@@ -5,110 +5,110 @@ slug: /cache_management
+The JuiceFS client automatically prefetches data into the cache based on the read pattern, thus improving sequential read performance. By default, it prefetches 1 block concurrently and caches it locally when reading. The local cache can be set up on any local file system based on HDD, SSD or memory.
 
 :::tip Hint
 You can check ["How JuiceFS Stores Files"](../reference/how_juicefs_store_files.md) to learn what a block is.
 :::
 
-The local cache can be adjusted at [mount file system](../reference/command_reference.md#juicefs-mount) with the following options.
+The local cache can be adjusted at [mounting file system](../reference/command_reference.md#juicefs-mount) with the following options.
 
 ```
 --cache-dir value         directory paths of local cache, use colon to separate multiple paths (default: "$HOME/.juicefs/cache" or "/var/jfsCache")
 --cache-size value        size of cached objects in MiB (default: 102400)
---free-space-ratio value  min free space (ratio) (default: 0.1)
+--free-space-ratio value  minimum free space (ratio) (default: 0.1)
 --cache-partial-only      cache only random/small read (default: false)
 ```
 
-Specifically, there are two ways if you want to store the local cache of JuiceFS in memory, one is to set `--cache-dir` to `memory` and the other is to set it to `/dev/shm/<cache-dir>`. The difference between these two approaches is that the former deletes the cache data after remounting the JuiceFS file system, while the latter retains it, and there is not much difference in performance between the two.
+Specifically, there are two ways to store the local cache of JuiceFS in memory -- setting `--cache-dir` to `memory` or `/dev/shm/<cache-dir>`. The difference between these two approaches is that, the former deletes the cache data after remounting the JuiceFS file system, while the latter retains it. There is not much difference in performance between them.
 
-The JuiceFS client writes data downloaded from the object store (including new uploads less than 1 block in size) to the cache directory as fast as possible, without compression or encryption. **Because JuiceFS generates unique names for all block objects written to the object store, and all block objects are not modified, there is no need to worry about invalidating the cached data when the file content is updated.**
+The JuiceFS client writes data downloaded from the object store (including new uploaded data containing less than 1 block) to the cache directory as fast as possible, without compression or encryption. **Because JuiceFS generates unique names for all block objects written to the object store, and all block objects are not modified, there is no need to worry about the cached data invalidation when the file content is updated.**
```

Comment:
```suggestion
The JuiceFS client writes data downloaded from the object storage (including new uploaded data containing less than 1 block) to the cache directory as fast as possible, without compression or encryption. **Because JuiceFS generates unique names for all block objects written to the object storage, and all block objects are not modified, there is no need to worry about the invalidation of the cached data when the file content is updated.**
```